lms/djangoapps/teams/tests/test_views.py

```diff
@@ -509,7 +516,7 @@ def test_expand_public_user(self):
         ('queryable', ['Search']),
         ('Tonga', ['Search']),
         ('Island', ['Search']),
-        ('search queryable', []),
+        ('not-a-query', []),
         ('team', ['Another Team', 'Public Profile Team']),
```

Comment:
One thing I'd like to see here is verification that `leaner_removed` events also get fired here. You may need to add those events in separately.